queries/dune_v2/period_slippage.sql

```diff
@@ -164,7 +164,7 @@ incoming_and_outgoing as (
         case
             when t.symbol = 'ETH' then 'WETH'
             when t.symbol is not null then t.symbol
-            else cast(t.token as varchar)
+            else cast(token as varchar)
```

Comment:
Regression? I would keep it as it was to stay consistent with the other columns, and I find this style more understandable.
```suggestion
            else cast(t.token as varchar)
```

Reply:
Unfortunately I don't know what this was doing before, but table t does not have token field.

I have not idea what was going on here before, but this definitely will not work:

```sql
incoming_and_outgoing as (
    SELECT
        block_time,
        tx_hash,
        dex_swaps,
        solver_address,
        case
            when t.symbol = 'ETH' then 'WETH'
            when t.symbol is not null then t.symbol
            else cast(token as varchar)
        end                                     as symbol,
          case
              when token = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
                  then 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
              else token
              end                                     as token,
          case
              when receiver = 0x9008d19f58aabd9ed0d60971565aa8510560ab41
                  then cast(cast(amount_wei as varchar) as int256)
              when sender = 0x9008d19f58aabd9ed0d60971565aa8510560ab41
                  then cast(-1 as int256) * cast(cast(amount_wei as varchar) as int256)
              end                                     as amount,
          transfer_type
    from batch_transfers i
        left outer join tokens.erc20 t
            on i.token = t.contract_address
            and blockchain = 'ethereum'
    where tx_hash not in (select tx_hash from excluded_batches)
)
```

Reply:
Interesting. Thanks for looking into that. Can we refer to the table where we are getting the symbol from then?

Reply:
```suggestion
            else cast(i.token as varchar)
```